### PR TITLE
Hello world project's pull request on html-css branch

### PR DESCRIPTION
**Pull request by adebsa2401**
As the last instruction of the Hello world project, I opened such pull request

### CODE REVIEW

File: hello.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse!</h1>
```
Comment (Reem-lab) [inline hello.html:1]: - [ ] I think you should create HTML code You can follow this [page](https://www.w3schools.com/html/html_editors.asp) so how to create HTML page and then you can write your heading inside it and don't forget to attach the stylesheet file so the chnages happens on the h1 tag element 🙏🏻🔥😊

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (Reem-lab) [inline test.md:1]: - [ ] Accodring to the requirements this file should have this text inside it so kindly add it 😊🔥🙏🏻


image

**Review by Reem-lab — COMMENTED** [review]
# Hi @adebsa2401, 👋🏻👋🏻

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Good commit messages ✅
- Descriptive pull request ✅

 ## Required Changes ♻️
## Kindly don't merge any pull request till it is aprroved by the code reviewer because if there is some changes to add so the new code should not be merged with the old code that's very important for GitHub flow 🙏🏻🔥😊🚀

_Check the comments under the review._

- [ ] Kindly ensure that your pull request has a descriptive title telling you what changes you have made so that reviewers and other people using your code will understand what the PR is all about. so the pull request it should be what you already did in this milestone so the title could be Hello Microverse and the PR more details about what you did 🚀😊💯🔥
- [ ] I think you didn't add the files for stylelint and webhint for testing the linters so kindly add them since they have the rules for organizing your code. and Don't foget to install the packages 🚀✅🙏🏻check this link [here](https://github.com/microverseinc/linters-config/tree/master/html-css)
image
## If you scroll down you can see how to install their packages then you can run commands to check if there is any error 🙏🏻🚀

- [ ] Please fix the stylelint linters by using this command `npx stylelint "**/*.{css,scss}"` you can see all the error in you style file then you can fix it, then run it again till it fixed 🙏🏻🔥🔥

image


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by adebsa2401** [comment]
Hi, Thanks for the review. So should I open a new pull request?

**Pull request merged**